app/views/holds/index.html.erb

```diff
@@ -1,78 +1,16 @@
 <h1>Holds</h1>
 
-<%= form_with url: '/holds/multiple', method: :delete do %>
-  <table class="table">
-    <caption>Holds that are ready for pickup</caption>
-    <thead>
-    <tr>
-      <th scope="col">
-        Select <br>
-        <%= check_box_tag(:ready_holds, '1', false, data: { select_all: 'ready' }) %>
-        <%= label_tag(:ready_holds, 'All') %>
-      </th>
-      <th scope="col">Title</th>
-      <th scope="col">Pickup at</th>
-      <th scope="col">Expires</th>
-    </tr>
-    </thead>
-    <tbody>
-    <% @holds_ready.each do |hold| %>
-      <tr>
-        <td>
-          <%= render_hold_select hold %>
-        </td>
-        <th scope="row">
-          <%= hold.title %><br>
-          <em><%= link_to 'Details', "https://catalog.libraries.psu.edu/catalog/#{hold.catkey}" %></em><br>
-          <%= hold.item_type_human %>
-        </th>
-        <td><%= hold.pickup_library_human %></td>
-        <td><%= render_expiration_date hold %></td>
-      </tr>
-    <% end %>
-    </tbody>
-  </table>
-  <%= submit_tag('Cancel Selected Holds') %>
-<% end %>
-
-<%= form_with url: '/holds/multiple', method: :delete do %>
-  <table class="table">
-    <caption>Holds that are not ready for pickup</caption>
-    <thead>
-
-    <tr>
-      <th scope="col">
-        Select <br>
-        <%= check_box_tag(:pending_holds, '1', false, data: { select_all: 'pending' }) %>
-        <%= label_tag(:pending_holds, 'All') %>
-      </th>
-      <th scope="col">Title</th>
-      <th scope="col">Availability</th>
-      <th scope="col">Pickup at</th>
-      <th scope="col">Expires</th>
-      <th scope="col">Status</th>
-    </tr>
-    </thead>
+<% if @holds_ready.blank? && @holds_not_ready.blank? %>
+<p>You have no holds at this time.</p>
+<% else %>
 
-    <tbody>
-    <% @holds_not_ready.each do |hold| %>
-      <tr>
-        <td>
-          <%= render_hold_select hold %>
-        </td>
-        <th scope="row">
-          <%= hold.title %><br>
-          <em><%= link_to 'Details', "https://catalog.libraries.psu.edu/catalog/#{hold.catkey}" %></em><br>
-          <%= hold.item_type_human %>
-        </th>
-        <td><%= render_queue_position hold %></td>
-        <td><%= hold.pickup_library_human %></td>
-        <td><%= l(hold.fill_by_date, format: :short) %></td>
-        <td><%= render_status hold %></td>
-      </tr>
-    <% end %>
-    </tbody>
+  <% if @holds_ready.present? %>
+    <%= render 'holds/holds_ready_table' %>
+  <% else %>
+    <p>You have no holds ready to be picked up at this time.</p>
+  <% end %>
+  <% if @holds_not_ready.present? %>
+    <%= render 'holds/holds_not_ready_table' %>
```

Comment:
@cdmo no else needed for this case?

Reply:
No, because if you don't have any "hold not yet ready to be picked up" this doesn't seem to me to rise to the occasion of "we should let you know." If you have no holds of either type (ready, not ready) a message is displayed. I was just trying to play the scenarios out in my head. We can ask Ruth to weigh in, I was just kinda guessing.

Reply:
OK, that makes sense to me, I think it is good like this. Ruth can review it on QA, would let us know if we need to let the user know :)